public/components/TagEdit/TagEdit.react.js

```diff
@@ -1,186 +1,243 @@
+import PaidContentInfoEdit from "./formComponents/paidcontent/PaidContentInfoEdit.react";
+import TrackingInfoEdit from "./formComponents/tracking/TrackingInformation.react.js";
+import CampaignInfoEdit from "./formComponents/campaigns/CampaignInformation.react.js";
 
-
-import * as tagTypes from '../../constants/tagTypes';
+import * as tagTypes from "../../constants/tagTypes";
 
 export default class TagEdit extends React.Component {
+  constructor(props) {
+    super(props);
 
-    constructor(props) {
-        super(props);
-
-        this.renderTagTypeSpecificFields.bind(this);
-    }
-
-    onUpdateSection(e) {
+    this.renderTagTypeSpecificFields.bind(this);
+  }
 
-      const sectionId = parseInt(e.target.value, 10);
-      const section = this.props.sections.filter((section) => section.id === sectionId)[0];
+  onUpdateSection(e) {
+    const sectionId = parseInt(e.target.value, 10);
+    const section = this.props.sections.filter(
+      section => section.id === sectionId
+    )[0];
 
-      this.props.updateTag(Object.assign({}, this.props.tag, {
+    this.props.updateTag(
+      Object.assign({}, this.props.tag, {
         section: sectionId,
         capiSectionId: section.path
-      }));
-    }
+      })
+    );
+  }
 
-    onUpdateCategory(selectedCategories) {
-      this.props.updateTag(Object.assign({}, this.props.tag, {
+  onUpdateCategory(selectedCategories) {
+    this.props.updateTag(
+      Object.assign({}, this.props.tag, {
         categories: selectedCategories
-      }));
-    }
+      })
+    );
+  }
 
-    onUpdateIsMicrosite(e) {
-      this.props.updateTag(Object.assign({}, this.props.tag, {
+  onUpdateIsMicrosite(e) {
+    this.props.updateTag(
+      Object.assign({}, this.props.tag, {
         isMicrosite: e.target.checked,
         section: undefined,
         capiSectionId: undefined
-      }));
-    }
-
-    renderTopicFields() {
-      return (
-        <div>
-          <div className="tag-edit__input-group" key="topic-section">
-            <label className="tag-edit__input-group__header">Section</label>
-              <input type="checkbox" checked={this.props.tag.isMicrosite} onChange={this.onUpdateIsMicrosite.bind(this)} disabled={this.props.pathLocked || !this.props.tagEditable}/>
-              <label className="tag-edit__label">is Microsite</label>
-              <SectionSelect
-                selectedId={this.props.tag.section}
-                sections={this.props.sections}
-                isMicrosite={this.props.tag.isMicrosite}
-                onChange={this.onUpdateSection.bind(this)}
-                disabled={this.props.pathLocked || !this.props.tagEditable}
-              />
-          </div>
+      })
+    );
+  }
+
+  renderTopicFields() {
+    return (
+      <div>
+        <div className="tag-edit__input-group" key="topic-section">
+          <label className="tag-edit__input-group__header">Section</label>
```

Comment:
unrelated to this PR, however we could add the [`for` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Attributes) to `label` to toggle the `input` on label click (bigger target area = better accessibility)